aerleon/lib/juniper.py

```diff
@@ -224,6 +224,7 @@ def __init__(
 
             # some options need to modify the actions
             self.extra_actions = []
+        self._ReplaceICMPv6()
```

Comment:
Crazy suggestion: does this need to be a separate method? It looks like lines 218-223 do something very similar here in `Term.__init__`, so it's probably fine to go with the flow and have the code live right here.